json.vim

```diff
@@ -0,0 +1,6 @@
+augroup FileType json
+  au!
+
+  " To indent json files on save
+  au FileType json autocmd BufWritePre <buffer> %!jq -S '.'
```

Comment:
I suggest you add some mechanism to check if `jq` is present or not in there before setting the auto command. 

See `:help executable` which tells you about `executable("ls")`. So you can then write: 

```vim
if executable("ls") 
  au FileType json autocmd BufWritePre <buffer> %!jq -S '.'
endif
```